### DISCUSSION TITLE
How to Add a custom HTML element inside a Cell?

### DISCUSSION THREAD

**Discussion by devadaskiran** [silevis/reactgrid#389] [Q&A]
How to Add a custom HTML element/ custom component inside a cell.

**Comment by qiufeihong2018** [comment]
Use the Custom Cell template to customize your cells

**Comment by qiufeihong2018** [comment]
https://reactgrid.com/docs/4.0/5-create-your-own-cell-template/

**Reply by devadaskiran** [reply]
Got it thanks
